tests/fixtures/admon.md

```diff
@@ -267,3 +267,28 @@ Does not render
 <p>!!!
 content</p>
 .
+
+
+
+MKdocs Closed Collapsible Sections
+.
+??? note
+    content
+.
+<div class="admonition note">
+<p class="admonition-title">Note</p>
+<p>content</p>
+</div>
```

Comment:
obviously this output HTML is not what you would actually want to use, since it does not contain any "information" that it should be collapsible and open/closed.
I think you just want this for mdformat, so don't need the HTML (just tokens) but I wonder if there could at least be a some "rudimentary" capture of this information here, e.g. something like a `collapsible` class and or `collapsible-open`/`collapsible-closed`?